### PR TITLE
Process instance signal popup UI review

### PR DESCRIPTION
**Pull request by cristianonicolai**
Adjusted form components class and size
 Removed extra OK button from UI.

### CODE REVIEW

**Comment by cristianonicolai** [comment]
Depends on https://github.com/uberfire/uberfire/pull/255

File: jbpm-console-ng-process-runtime/jbpm-console-ng-process-runtime-client/src/main/java/org/jbpm/console/ng/pr/client/editors/instance/signal/ProcessInstanceSignalViewImpl.html [hunk 1]
```diff
@@ -1,26 +1,26 @@
...
+    <div>
+        <div class="form-horizontal">
+            <fieldset>
+                <div class="form-group">
+                    <label class="control-label col-md-3" for="signalRefText" data-field="signalRefLabel"></label>
 
-                <div class="col-md-10">
-                    <input type="text" data-field="signalRefText" id="signalRefText" placeholder="" class="form-control"/>
+                    <div class="col-md-9">
+                        <input type="text" data-field="signalRefText" id="signalRefText" placeholder="" class="form-control"/>
```
Comment (csadilek) [inline jbpm-console-ng-process-runtime/jbpm-console-ng-process-runtime-client/src/main/java/org/jbpm/console/ng/pr/client/editors/instance/signal/ProcessInstanceSignalViewImpl.html]: just a note. it's not required to use a data-field, just id is fine. @Inject @DataField will work for id, data-field, and class.

**Comment by csadilek** [comment]
+1

**Comment by csadilek** [comment]
master: 0147d1a21bda4a4ba07f052f2f6878b3ae9d9a71

**Pull request closed without merging**